gensim/models/word2vec.py

```diff
@@ -79,6 +79,7 @@
 import itertools
 
 from gensim.utils import keep_vocab_item
+from keyedvectors import KeyedVectors
```

Comment:
Relative imports don't play well with python3, and gensim seems to use absolute imports everywhere